.github/workflows/go.yml

```diff
@@ -1,39 +0,0 @@
-# This workflow will build a golang project
```

Comment:
Is deleting these github actions workflows intentional?

Reply:
Yep! They were broken and needed updating. I'll bring them back soon, promise.